### PR TITLE
Publish confirm add readme

### PR DESCRIPTION
**Pull request by cachrisman**

### CODE REVIEW

File: samples/publish-confirm/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Publish Confirmation UI Extension
+
+This is a sidebar UI extension which shows the entry status and a publish button. It also has a like to unpublish the entry and shows when the entry was last saved.
+
+![image](https://user-images.githubusercontent.com/6521666/60894265-a3cb1180-a262-11e9-96d3-0f5ffaeedbb7.png)
```
Comment (jelz) [inline samples/publish-confirm/README.md]: Please commit images to the repo

File: samples/publish-confirm/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Publish Confirmation UI Extension
+
+This is a sidebar UI extension which shows the entry status and a publish button. It also has a like to unpublish the entry and shows when the entry was last saved.
```
Comment (floelhoeffel) [inline samples/publish-confirm/README.md]: like = link ?

File: samples/publish-confirm/README.md (hunk 1 above)
Comment (cachrisman) [reply samples/publish-confirm/README.md]: done!

File: samples/publish-confirm/README.md (hunk 2 above)
Comment (cachrisman) [reply samples/publish-confirm/README.md]: done

**Pull request merged**